src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/AreComObjectsAvailableForCleanupTests.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public class AreComObjectsAvailableForCleanupTests
```

Comment:
Thank you! I can remove this from coreclr now 😄

Reply:
Woo!